endpoints/api_config.py

```diff
@@ -2149,7 +2166,9 @@ def get_descriptor_defaults(self, api_info, hostname=None):
         'extends': 'thirdParty.api',
         'root': '{0}://{1}/{2}'.format(protocol, hostname, base_path),
         'name': api_info.name,
-        'version': api_info.version,
+        'version': api_info.api_version,
```

Comment:
is this `version` field still needed?

Reply:
_Probably_ not, but I can't rule out there being something stupid that'll break if it's gone.